src/frontends/tensorflow/src/input_model.cpp

```diff
@@ -175,7 +175,15 @@ void InputModel::InputModelTFImpl::load_places() {
             }
             std::vector<std::string> names = {op_name};
             auto tensor_place = std::make_shared<TensorPlace>(m_input_model, pshape, type, names);
+
+            // In Model Optimizer user can refer to model inputs by a name of Placeholder
+            // and its output port, for example, using `input_name` and `input_name:0`
+            // Also, SavedModel format contains a signature that maps external input names to internal ones with port
+            // index like `input_name` maps to `serving_default_input_name:0`.
+            // So we have to store with `:0` as well to get its tensor by `get_place_by_tensor_name` method
             m_tensor_places[op_name] = tensor_place;
+            m_tensor_places[op_name + ":0"] = tensor_place;
```

Comment:
I didn't see such models. I'm not sure it is possible. Just FYI, something strange may be in case op_name already contains :0

Reply:
It is added because internal name for original inputs `serving_default_input:0`. Namely, it goes with suffix `:0`.
So for them we create new tensor places always, though, tensor places were created for them in `load_places` method.